snippets/functions/third-party/manifest.json

```diff
@@ -20,6 +20,30 @@
       "title": "Using node-fetch dependency",
       "snippet": "/nodeFetch.js",
       "description": "Making HTTP requests with node-fetch dependency. Atlas App Services does not support v3 of node-fetch, use v2 instead."
+    },
+    {
+      "id": "fdf44706-adbe-4880-87b5-e9c28f6110af",
+      "title": "Using AWS SES dependency",
+      "snippet": "/awsSES.js",
+      "description": "Sending an email with AWS SES. To authenticate with the AWS client, store your AccessKeyID and SecretAccessKey as values. Note: This is using AWS SDK v3."
+    },
+    {
+      "id": "fdf44706-adbe-4880-87b5-e9c28f6110ag",
+      "title": "Using AWS Kinesis dependency",
+      "snippet": "/awsKinesis.js",
+      "description": "Retrieving stream details with AWS Kinesis. To authenticate with the AWS client, store your AccessKeyID and SecretAccessKey as values. Note: This is using AWS SDK v3."
+    },
+    {
+      "id": "fdf44706-adbe-4880-87b5-e9c28f6110aa",
+      "title": "Retrieving from AWS S3 bucket",
+      "snippet": "/awsS3GetObject.js",
+      "description": "Retrieving from AWS S3 bucket with the bucket name and key. To authenticate with the AWS client, store your AccessKeyID and SecretAccessKey as values. Note: This is using AWS SDK v3."
+    },
+    {
+      "id": "fdf44706-adbe-4880-87b5-e9c28f6110ab",
+      "title": "Putting object in AWS S3 bucket",
+      "snippet": "/awsS3PutObject.js",
+      "description": "Putting an object into AWS S3 bucket with the bucket name and key. To authenticate with the AWS client, store your AccessKeyID and SecretAccessKey as values. Note: This is using AWS SDK v3."
```

Comment:
Up to you but we are also storing the bucket key in values, not really necessary to call it out but wanted to make sure it's intentional.